packages/taxdat/R/plot_MCMC.R

```diff
@@ -27,10 +27,12 @@ plot_chain_convergence <- function(name,
   
   time_effect_param <- tolower(yaml::read_yaml(paste0(cholera_directory,"/",config))$time_effect)
   if(time_effect_param == 'no' | time_effect_param == 'false'){
-    pars <- pars[pars != "eta"]
+    warning("There is no time effect, meaning eta and alpha will not be shown. ")
```

Comment:
There may still be an alpha even if the time_effect is off. We specify whether an intercept should be in the model separately